org.apache.netbeans.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "org.apache.netbeans",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk9"
```

Comment:
Probably works with 10, never tested though. There's no info if it works with 11.

Reply:
The next version (10.0) is a work in progress. I would use openjdk11 once it is released.

Reply:
I did a quick exploratory test with openjdk10 and I found no issues. Since nb9 is declared to support Java 10 (https://netbeans.apache.org/download/nb90/index.html)  I changed the sdk-extension to openjdk10 and the .desktop file to make nb use it.